addons/folo/common/src/main/java/org/commonjava/indy/folo/data/FoloRecordCache.java

```diff
@@ -15,41 +15,28 @@
  */
 package org.commonjava.indy.folo.data;
 
-import org.commonjava.cdi.util.weft.ExecutorConfig;
-import org.commonjava.indy.folo.conf.FoloConfig;
-import org.commonjava.indy.folo.data.idxmodel.AffectedStoreRecordKey;
-import org.commonjava.indy.folo.model.AffectedStoreRecord;
+import org.commonjava.indy.IndyWorkflowException;
+import org.commonjava.indy.content.ContentManager;
+import org.commonjava.indy.content.DownloadManager;
```

Comment:
Seems not needed. Forgot to remove after extract these to FoloTrackingListner. Will fix these.